Tools/Scripts/libraries/resultsdbpy/resultsdbpy/bug_trackers/bugzilla.py

```diff
@@ -0,0 +1,117 @@
+from urllib.parse import quote
+from resultsdbpy.bug_trackers.bug_description import translate_selected_dots_to_bug_title_and_description
+from resultsdbpy.controller.bug_tracker_controller import BugTrackerConfig
+
+class Bugzilla(BugTrackerConfig):
```

Comment:
This isn't "Bugzilla" in general though, right? It's WebKit bugzilla specifically?